seleniumByTechnocredits/src/seleniumPractise/pallavi/Registration_Validation.java

```diff
@@ -0,0 +1,373 @@
+		} 
+		else 
+		{
+			reg.alertMessage();
+		}
+		al.accept();
+		driver.close();
+	}
+
+	void alertMessage() throws InterruptedException 
+	{
+		Alert alt = driver.switchTo().alert();
+		Thread.sleep(2000);
+		if (alt.getText().equals(fullNameMessage)) 
+		{
+			System.out.println("Correct Message is displaying as: " + alt.getText());
+		} 
+		else if (alt.getText().equals(addressMessage)) 
+		{
+			System.out.println("Correct Message is displaying as: " + alt.getText());
+		} 
+		else if (alt.getText().equals(eMailMessage)) 
+		{
+			System.out.println("Correct Message is displaying as: " + alt.getText());
+		} 
+		else if (alt.getText().equals(cityMessage)) 
+		{
+			System.out.println("Correct Message is displaying as: " + alt.getText());
+		} 
+		else if (alt.getText().equals(companyMessage)) 
+		{
+			System.out.println("Correct Message is displaying as: " + alt.getText());
+		} 
+		else if (alt.getText().equals(userNameMessage)) 
+		{
+			System.out.println("Correct Message is displaying as: " + alt.getText());
+		} 
+		else if (alt.getText().equals(passwordMessage)) 
+		{
+			System.out.println("Correct Message is displaying as: " + alt.getText());
+		} 
+		else if (alt.getText().equals(reenterPasswordMessage)) 
+		{
+			System.out.println("Correct Message is displaying as: " + alt.getText());
+		} 
+		else if (alt.getText().equals("Please agree to terms of service and privacy policy")) 
+		{
+			System.out.println("Correct Message is displaying as: " + alt.getText());
+		} 
+		else 
+		{
+			System.out.println("Displaying message is not the expected one: " + alt.getText());
+
+		}
+		alt.accept();
+	}
+
+	void scroll() 
+	{
+		ele = driver.findElement(By.id("btnsubmitsignUp"));
+		JavascriptExecutor js = (JavascriptExecutor) driver;
+		js.executeScript("arguments[0].scrollIntoView(true)", ele);
+	}
+
+	void blankDetails() throws InterruptedException 
+	{
+		scroll();
+		ele.click();
+		alertMessage();
+	}
+
+	void fullN() throws InterruptedException 
+	{
+		WebElement fName = driver.findElement(By.id("fullName"));
+		fName.sendKeys(fullName);
+		scroll();
+		ele.click();
+		alertMessage();
+	}
+
+	void addressN() throws InterruptedException 
+	{
+		WebElement fName = driver.findElement(By.id("fullName"));
+		fName.clear();
+		fName.sendKeys(fullName);
+		WebElement aName = driver.findElement(By.id("address"));
+		aName.sendKeys(address);
+		scroll();
+		ele.click();
+		alertMessage();
+	}
+
+	void emailN() throws InterruptedException 
+	{
+		WebElement fName = driver.findElement(By.id("fullName"));
+		fName.clear();
+		fName.sendKeys(fullName);
+		WebElement aName = driver.findElement(By.id("address"));
+		aName.clear();
+		aName.sendKeys(address);
+		WebElement eName = driver.findElement(By.id("useremail"));
+		eName.sendKeys(email);
+		scroll();
+		ele.click();
+
+		Alert alt = driver.switchTo().alert();
+		Alert alt1 = driver.switchTo().alert();
+		Alert alt2 = driver.switchTo().alert();
```

Comment:
Hi Pallavi,

Also adding to Prachi's  review comment, alt2 object of alert is not used to validate any alert message.